user_form_wrapper.html

```diff
@@ -190,7 +189,7 @@
         if ('{{ page.custom_fields.form_label_text_sms_optin }}') {
           smsOptInLanguage = '{{ page.custom_fields.form_label_text_sms_optin }}'
         }
-        jQuery('.mobile_phone_box').after(jQuery('<div id="id_user_mobile_subscriber_box" class="input-checkbox ak-input-type-custom  ak-err-below"><span class="ak-userfield-checkbox-set"><label class="ak-userfield-checkbox-choice"><input type="checkbox" name="user_mobile_subscriber" id="id_user_mobile_subscriber_true" class="ak-userfield-input" value="true"> '+smsOptInLanguage+'</label> </span></div>'));
+        jQuery('.mobile_phone_box').after(jQuery('<div id="id_user_mobile_subscriber_box" class="input-checkbox ak-input-type-custom ak-err-below user_mobile_subscriber_box"><span class="ak-userfield-checkbox-set"><label class="ak-userfield-checkbox-choice"><input type="checkbox" name="user_mobile_subscriber" id="id_user_mobile_subscriber_true" class="ak-userfield-input" value="true"> '+smsOptInLanguage+'</label> </span></div>'));

```

Comment:
@sukhada I'm confused why this fixed if this class wasn't present previously. I want to be sure I understand so we don't miss anything else - was this related to going from an ID in this jQuery function to a class, and if so, why did it cause this to fail?

Reply:
@cjmabry Yes, here: https://github.com/350org/ak_intl_v3/commit/1b514c7a517b8af0c9e350676af3b7def7f8b80e#diff-da63cec53b6ab5cbe60ca040f7096aaf88c28015df0f9d1d5ae57e514132b6caR42. I was assuming that this code would have added the class properly to the mobile phone field, but it looks like it was a special case - meaning that we have to add the class manually so that all the code that's using the selector `.user_mobile_subscriber_box` works

Reply:
I think I understand, thanks for explaining. What was the reasoning for moving from IDs to classes here? It seems like a lot of javascript depends on those IDs, so I want to be sure nothing else is behaving out of order.

Reply:
According to AK support, this was an incorrect naming convention, which was leading to bugs (like always visible fields not showing up on forms). The `actionkit.js` code looks for IDs/classes in the form `ak-fieldbox-{{ field.field_name }}`, which is why we were seeing these errors.

Reply:
Got it! Thanks for explaining.